packages/exhibitor/extra/start_exhibitor.py

```diff
@@ -49,6 +54,7 @@ def invoke_detect_ip():
     '-Djava.util.prefs.userRoot=/var/lib/dcos/exhibitor/',
     '-Duser.home=/var/lib/dcos/exhibitor/',
     '-Duser.dir=/var/lib/dcos/exhibitor/',
+    '-Djna.tmpdir=%s' % jna_tmpdir,
```

Comment:
I should have worded my question differently.

ZK will automatically look for that environment variable when it's start script runs, however the value of that variable is being set to the exact same value we're using for exhibitor. 

I could envision something like this in the unit or environment file
```bash
JNA_TMP_DIR=/var/lib/dcos/exhibitor/tmp
SERVER_JVMFLAGS="-Djna.tmpdir=$JNA_TMP_DIR"
```
Then in `start_exhibitor.py`
```python
'-Djna.tmpdir=%s' % os.environ['JNA_TMP_DIR']
```

Doing it this way keeps all the grep points, with a single definition that if (when?) has to be updated touches everything.